### PR TITLE
Update spawn_star.inc.c

### PR DESCRIPTION
**Pull request by PyroJay4**
ahahahahahahahahahahahahahahahahahahaha

### CODE REVIEW

File: src/game/behaviors/spawn_star.inc.c [hunk 1]
```diff
@@ -79,7 +79,7 @@ void bhv_star_spawn_loop(void) {
             o->oFaceAngleYaw += 0x1000;
             spawn_object(o, MODEL_NONE, bhvSparkleSpawn);
             cur_obj_play_sound_1(SOUND_ENV_STAR);
-            if (o->oTimer == 30) {
+            if (o->oTimer == 29) {
```
Comment (thecozies) [inline src/game/behaviors/spawn_star.inc.c:82]:
```suggestion
            if (o->oTimer == 69) {
```

**Review by thecozies — APPROVED** [review]
looks goooooooooooooood

**Review by gheskett — APPROVED** [review]
Yeah, ship it

**Comment by Reonu** [comment]
why master

but fine I guess

**Pull request merged**